.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.phar
 composer.lock
 vendor
+.idea/
```

Comment:
This should be in your global `.gitignore`, and doesn't belong in the package 🙂 Besides, it's out of the scope for the feature you're requesting.